src/pages/home/ReportScreen.js

```diff
@@ -235,10 +235,7 @@ class ReportScreen extends React.Component {
         const addWorkspaceRoomErrors = lodashGet(this.props.report, 'errorFields.addWorkspaceRoom');
         const hideComposer = isArchivedRoom || !_.isEmpty(addWorkspaceRoomErrors);
         return (
-            <ScreenWrapper
-                style={[styles.appContent, styles.flex1, {marginTop: this.state.viewportOffsetTop}]}
-                keyboardAvoidingViewBehavior={Platform.OS === 'android' ? '' : 'padding'}
-            >
+            <ScreenWrapper style={[styles.appContent, styles.flex1, {marginTop: this.state.viewportOffsetTop}]}>
```

Comment:
Same here, this is simply reverting [this PR](https://github.com/Expensify/App/pull/10567/files)